kotlinx.fuzz.test/settings.gradle.kts

```diff
@@ -1,3 +1,7 @@
 pluginManagement {
     includeBuild("..")
+    repositories {
+        mavenCentral()
+        maven(url = "https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies")
```

Comment:
Do I understand correctly that now all users of our plugin will have to add this additional plugin management repo to their build?

Reply:
Unfortunately, yes and to the `repositories` in `build.gradle.kts`